JS-Algorithms/challenges.js

```diff
@@ -14,6 +14,15 @@ Invoking "readableTime(3690)" should return "01:01:30" (HH:MM:SS)
 
 const readableTime = (seconds) => {
   // YOUR CODE HERE...
+  if( isNaN(seconds) || seconds < 0) return null;
+
+  const hours = Math.floor(seconds / 3600);
+  const minutes = Math.floor((seconds - (hours * 3600)) / 60 );
+  const second = seconds - (hours * 3600) - (minutes * 60);
```

Comment:
in this code I see you have 3600 and 60, to me this are your conversion factors, maybe it would be good to put this into a variable so you can easily change, reuse, etc